plugins/postgres/dbt/adapters/postgres/relation_configs/materialized_view.py

```diff
@@ -66,17 +66,19 @@ def from_dict(cls, config_dict: dict) -> "PostgresMaterializedViewConfig":
         return materialized_view
 
     @classmethod
-    def from_model_node(cls, model_node: ModelNode) -> "PostgresMaterializedViewConfig":
-        materialized_view_config = cls.parse_model_node(model_node)
+    def from_relation_config(
+        cls, relation_config: RelationConfig
+    ) -> "PostgresMaterializedViewConfig":
+        materialized_view_config = cls.parse_model_node(relation_config)
         materialized_view = cls.from_dict(materialized_view_config)
         return materialized_view
 
     @classmethod
-    def parse_model_node(cls, model_node: ModelNode) -> dict:
-        indexes: List[dict] = model_node.config.extra.get("indexes", [])
+    def parse_model_node(cls, relation_config: RelationConfig) -> dict:
```

Comment:
Return type should be `Dict[str, Any]`.

This should be renamed to `parse_relation_config`/`parse_config` to match `from_relation_config`/`from_config` above.